src/main/java/jmgomez/apipolicy/mapper/MapStructService.java

```diff
@@ -0,0 +1,27 @@
+package jmgomez.apipolicy.mapper;
+
+import jmgomez.apipolicy.model.Accident;
+import jmgomez.apipolicy.model.Policy;
+import jmgomez.apipolicy.model.dto.AccidentDto;
+import jmgomez.apipolicy.model.dto.PolicyDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MapStructService {
```

Comment:
??? why do you need a service to expose the mapper? It's already an available component. Which kind of improvement or need covers this? Have you read the article I've sent you and you've seen a service there?